playbook/app/pb_kits/playbook/utilities/_align_content.scss

```diff
@@ -1,29 +1,10 @@
-.align_content_start {
-  display: flex;
-  align-content: start !important;
-}
-
-.align_content_end {
-  display: flex;
-  align-content: end !important;
-}
-
-.align_content_center {
-  display: flex;
-  align-content: center !important;
-}
-
-.align_content_space_between {
-  display: flex;
-  align-content: space-between !important;
-}
-
-.align_content_space_around {
-  display: flex;
-  align-content: space-around !important;
-}
-
-.align_content_space_evenly {
-  display: flex;
-  align-content: space-evenly !important;
-}
\ No newline at end of file
+$align_content_values: (
```

Comment:
@RobGentile17 you're gonna love all this cleanup and simplification!

Reply:
I can't wait for it to be live and in use!